File: JAVA/chapter6_examples/chapter6_examples/TestMax.java
/*
 * Demonstrates defining and calling a method max to return the largest of two int values
 */

package chapter6_examples;

public class TestMax {
	// main method	
	public static void main(String[] args) {
		int i = 5;
		int j = 2;
		int k = max(i, j); // Invoke method max

		System.out.println("The max is " + k);
	}

	// A method that returns the max value of two integers
	public static int max(int num1, int num2) {
		int result; 

		if (num1 > num2)
			result = num1;
		else
			result = num2;

		return result;
	}
}

File: Python Basic/TestMax.py
# Return the max between two numbers 
def max(num1, num2): 
    if num1 > num2:
        result = num1
    else:
        result = num2

    return result

def main():
    i = 5
    j = 2
    k = max(i, j) # Call the max function
    print("The maximum between", i, "and", j, "is", k)

main() # Call the main function
